server.js

```diff
@@ -14,6 +14,7 @@
 // Require Employee Schema from Database
   var employee = require("./models/Employee");
   var EmployeeSchedule = require("./models/employeeSchedule");
+  var announcements = require("./models/announcements");
```

Comment:
@mr-attack @clsavino @hcbreedl Hmm this must have snuck past me in previous commits. All models should be moved out of server and into controllers.  <That's where the db routes are.